plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/ui/search/SearchView.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * 
+ */
+package org.jboss.reddeer.eclipse.ui.search;
+
+import org.jboss.reddeer.core.exception.CoreLayerException;
+import org.jboss.reddeer.core.matcher.WithTextMatcher;
+import org.jboss.reddeer.swt.api.Tree;
+import org.jboss.reddeer.swt.impl.link.DefaultLink;
+import org.jboss.reddeer.swt.impl.tree.DefaultTree;
+import org.jboss.reddeer.workbench.impl.view.WorkbenchView;
+
+/**
+ * Represents Search view 
+ * 
+ * @author odockal
+ *
+ */
+public class SearchView extends WorkbenchView {
+
+	/**
+	 * Constructs Search view object
+	 */
+	public SearchView() {
+		super("Search");
+	}
+	
+	/**
+	 * Says whether searching of referenced item was successful or not.
+	 * @return true if results are not present in Search view, false otherwise
+	 */
+	public boolean isSearchEmpty() {
+		activate();
+		try {
+			new DefaultLink(new WithTextMatcher(
+					"No search results available. Start a search from the <a>search dialog</a>..."));
+			return true;
+		} catch (CoreLayerException e){
+			return false;
+		}
+	}
+
+	/**
+	 * Returns all search results from Search view as a Tree object with default search option setup
+	 * @return Tree object of search results
+	 */
+	public Tree getSearchResultsAsTree() {
```

Comment:
we are trying to avoid returning swt-layer objects in eclipse layer. I would be better if you implement a new object something like a "SearchResult" which would wrap a single tree item and we would be able to also wrap other methods in the future. 

So this method would return List<SearchResult> and i would call it getSearchResults()